src/Workspaces/CSharp/Portable/AddImports/CSharpAddImportsService.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
See comments on IAddImportsService.

Reply:
There wasn't anything to do here during the fan-out from https://github.com/dotnet/roslyn/pull/37512#discussion_r314099547.